Create password on demand

f-bader opened the pull request:
Do not store the password in cleartext. Since the password is temporary it can be generated on runtime.

azurecla commented:
Hi __@deneha__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

f-bader committed:
Create password on demand

Do not store the password in cleartext. Since the password is temporary it can be generated on runtime.
